enhancements/security/security-profiles-operator.md

```diff
@@ -0,0 +1,629 @@
+---
+title: security-profiles-operator
+authors:
+  - "@jaormx"
+  - "@jhrozek"
+  - "@saschagrunert"
+reviewers:
+  - "@ashcrow"
+  - "@mrunalp"
+approvers:
+  - TBD
+  - "@oscardoe"
```

Comment:
this needs the auth team as approver as they own SCCs. Also OLM team must be here.

Reply:
will add, thanks!